lib/bots/versionbot.ts

```diff
@@ -331,8 +377,18 @@ export class VersionBot extends GithubBot.GithubBot {
             }).then(() => {
                 this.log(ProcBot.LogLevel.INFO, `Upped version of ${repoFullName} to ${newVersion}; ` +
                 `tagged and pushed.`);
+            }).catch((err: Error) => {
+                // Call the VersionBot error specific method.
+                this.reportError({
+                    brief: `Versionbot failed to create a new version ready for merge for ${repoFullName}#${pr.number}`,
```

Comment:
nit: I find this message a bit hard to understand, can we rephrase these messages to something simpler like

`Versionbot failed to merge ${repoFullName}#${pr.number}`

After all we don't know which operation failed